src/govuk/components/pagination/_index.scss

```diff
@@ -0,0 +1,181 @@
+@include govuk-exports("govuk/component/pagination") {
+  .govuk-pagination {
+    border-top: 1px solid $govuk-border-colour;
```

Comment:
@timpaul why is there a top border in the component? I don't think it's needed. It also feels like something that would be frequently overridden depending on the needs of the service.